packages/code.rb

```diff
@@ -35,13 +33,15 @@ class Code < Package
   ENV['PATH'] = "#{ENV['HOME']}/.nodebrew/current/bin:#{ENV['PATH']}"
 
   def self.build
-    node_old = `nodebrew ls | fgrep 'current: ' | cut -d' ' -f2`
+    node_old = `nodebrew ls | fgrep 'current: ' | cut -d' ' -f2 | grep -v none`
     system "nodebrew install 8.15.0 || true"
     system "nodebrew use 8.15.0"
     system "npm install gulp"
     system "yarn install"
     system "yarn run gulp vscode-linux-#{@arch}"
-    system "nodebrew use #{node_old}"
+    if node_old != ''
+      system "nodebrew use #{node_old}"
+    end
```

Comment:
This is not going to work.  node_old will always be not equal to '' since you didn't chomp.

Reply:
Have you even tried building binaries?  Apparently, not since `nodebrew use none` always errors out.

Reply:
So all I need to do is add `.chomp` at the end of the line that assigns `#{node_old}`, right?

Reply:
Anytime you use commands with backticks, you need to chomp the end to get rid of the non-printable characters.

Reply:
Got it. So is that ALL I need to do?

Reply:
Test it without any node versions installed and see what happens.

Reply:
Done.